### PR TITLE
Elmining consultation was implemented according to the ID field.

### PR DESCRIPTION
**Pull request by TheYisusGithub**

### CODE REVIEW

File: src/main/java/com/fundation/logic/database/DBConnection.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import java.sql.SQLException;
+import java.sql.Statement;
+/**
+ * Implements insert in a table .
+ *
+ * @author Andres Burgos, Jesus Menacho
+ * @version 1.0
+ */
+public class DBConnection {
+
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/database/DBConnection.java]: unnecessary line

File: src/main/java/com/fundation/logic/database/DBConnection.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            dbcon = new DBConnection();
+        }
+        return dbcon;
+    }
+
+    private static void initConnection(){
+        String file="AT10-player.db";
+        String url ="jdbc:sqlite:" + file;
+        try {
+            conn=DriverManager.getConnection(url);
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/database/DBConnection.java]: space before and after =

File: src/main/java/com/fundation/logic/database/DBConnection.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+            statement.execute(sql);
+        } catch (SQLException e) {
+            e.getMessage();
+        }
+
+    }
+    public Connection getConnection(){
+        return conn;
+    }
+}
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/database/DBConnection.java]: one line missing

File: build.gradle [hunk 4]
```diff
@@ -9,4 +9,5 @@ dependencies {
     compile group: 'uk.co.caprica', name: 'vlcj', version: '3.12.1'
     testCompile group: 'junit', name: 'junit', version: '4.12'
     testImplementation 'junit:junit:4.12'
+    compile group: 'org.xerial', name: 'sqlite-jdbc', version: '3.27.2.1'
 }
```
Comment (titor-oopart) [inline build.gradle:13]: add a empty line

File: src/main/java/com/fundation/logic/database/DBConnection.java [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+ *
+ * @author Andres Burgos, Jesus Menacho
+ * @version 1.0
+ */
+public class DBConnection {
+
+    private static DBConnection dbcon = new DBConnection();
+    private static Connection conn;
+
+    private DBConnection(){
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/database/DBConnection.java]: put a space between  ")" "{"

File: src/main/java/com/fundation/logic/database/DBConnection.java [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+        initConnection();
+    }
+
+    public static DBConnection getInstance(){
+        if (dbcon == null){
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/database/DBConnection.java]: put a space after ")"

File: src/main/java/com/fundation/logic/database/DBConnection.java [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+            dbcon = new DBConnection();
+        }
+        return dbcon;
+    }
+
+    private static void initConnection(){
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/database/DBConnection.java]: put a space after ")"

File: src/main/java/com/fundation/logic/database/DBConnection.java [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+        String file="AT10-player.db";
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/database/DBConnection.java]: put a space before and after "="

File: src/main/java/com/fundation/logic/database/DBConnection.java [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+        String url ="jdbc:sqlite:" + file;
+        try {
+            conn=DriverManager.getConnection(url);
+        } catch (SQLException e) {
+            e.getMessage();
+        }
+        String sql =    "CREATE TABLE IF NOT EXISTS criterias (\n"
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/database/DBConnection.java]: delete the unnecessary space after "="

File: src/main/java/com/fundation/logic/database/DBConnection.java [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+        try {
+            Connection connection = conn;
+            Statement statement = connection.createStatement();
+            statement.execute(sql);
+        } catch (SQLException e) {
+            e.getMessage();
+        }
+
+    }
+    public Connection getConnection(){
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/database/DBConnection.java]: put a space after ")"

**Pull request closed without merging**